style.css

```diff
@@ -2,4 +2,141 @@
   margin: 0;
   padding: 0;
   font-family: Arial, Helvetica, sans-serif;
-}
\ No newline at end of file
+}
+.container {
+  background-image: linear-gradient(190deg, #f5fafb 47.27%, #e9f6f9 85.77%);
+  border-radius: 3.5rem;
+  height: 23rem;
+  width: 80rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 3.13rem 2.6rem;
+}
+.main {
+  padding: 2rem;
+  width: 56rem;
+  border-radius: 16px;
+  border: 1px solid #eaecf0;
+  height: 75%;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  background-image: linear-gradient(
+    180deg,
+    #fff 0%,
+    rgba(255, 255, 255, 0.5) 99.5%
+  );
+  box-shadow: 0 0.882px 1.568px 0 rgba(38, 44, 59, 0.02),
+    0 2.119px 3.767px 0 rgba(38, 44, 59, 0.03),
+    0 3.99px 7.093px 0 rgba(38, 44, 59, 0.03),
+    0 7.117px 12.653px 0 rgba(38, 44, 59, 0.04),
+    0 13.312px 23.666px 0 rgba(38, 44, 59, 0.05),
+    0 31.865px 56.648px 0 rgba(38, 44, 59, 0.07);
+}
+
+.info {
```

Comment:
make it flexible,
use flex:1 on both children, not fixed widths on children or textarea